competition_inputs/BottleJSONReader.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+# G J 2-7-2023
+# Opens and imports JSON File 'BottleJSON.json' as a list
+f = open("competition_inputs/BottleJSON.json")
+data = json.load(f)
+
+# Defines Array of Bottles from the list from the JSON file
+Bottles = [
```

Comment:
I think since the info loaded from the json file will always have the same format/types, you should create a `TypedDict` to better organize the data once it is loaded in.